### ISSUE TITLE
Build Error on Android

### ISSUE THREAD

**Issue by martnst** [rumax/react-native-PDFView#61]
I am getting the following error after: 

- `$ yarn add react-native-view-pdf`
- `$ react-native link react-native-view-pdf`
- update Andoird minSdkVersion 19 

Error Log: 
```
 Task :app:transformDexArchiveWithDexMergerForDebug FAILED
D8: Program type already present: com.reactlibrary.BuildConfig
java.lang.RuntimeException: com.android.builder.dexing.DexArchiveMergerException: Error while merging dex archives: ./android/app/build/intermediates/transforms/dexBuilder/debug/83, ./android/app/build/intermediates/transforms/externalLibsDexMerger/debug/0, ./android/app/build/intermediates/transforms/dexBuilder/debug/53.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/54.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/55.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/56.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/57.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/58.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/60.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/63.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/64.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/65.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/66.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/67.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/69.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/70.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/71.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/72.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/74.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/75.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/77.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/78.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/79.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/80.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/81.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/82.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/100.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/102.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/104.jar
        at java.util.concurrent.ForkJoinTask$AdaptedCallable.exec(ForkJoinTask.java:1431)
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
        at java.util.concurrent.ForkJoinTask.externalAwaitDone(ForkJoinTask.java:326)
        at java.util.concurrent.ForkJoinTask.doJoin(ForkJoinTask.java:391)
        at java.util.concurrent.ForkJoinTask.join(ForkJoinTask.java:719)
        at com.google.common.collect.ImmutableList.forEach(ImmutableList.java:397)
        at com.android.build.gradle.internal.transforms.DexMergerTransform.transform(DexMergerTransform.java:221)
        at com.android.build.gradle.internal.pipeline.TransformTask$2.call(TransformTask.java:221)
        at com.android.build.gradle.internal.pipeline.TransformTask$2.call(TransformTask.java:217)
        at com.android.builder.profile.ThreadRecorder.record(ThreadRecorder.java:102)
        at com.android.build.gradle.internal.pipeline.TransformTask.transform(TransformTask.java:212)
        at sun.reflect.GeneratedMethodAccessor368.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
        at org.gradle.api.internal.project.taskfactory.IncrementalTaskAction.doExecute(IncrementalTaskAction.java:46)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:39)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:26)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:121)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:199)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:110)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:110)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:92)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:70)
        at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
        at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:62)
        at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:60)
        at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:97)
        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:87)
        at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:52)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:52)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker$1.run(DefaultTaskGraphExecuter.java:248)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:199)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:110)
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:241)
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:230)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.processTask(DefaultTaskPlanExecutor.java:123)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.access$200(DefaultTaskPlanExecutor.java:79)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:104)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:98)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.execute(DefaultTaskExecutionPlan.java:626)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.executeWithTask(DefaultTaskExecutionPlan.java:581)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.run(DefaultTaskPlanExecutor.java:98)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
        at java.lang.Thread.run(Thread.java:748)
Caused by: com.android.builder.dexing.DexArchiveMergerException: Error while merging dex archives: ./android/app/build/intermediates/transforms/dexBuilder/debug/83, ./android/app/build/intermediates/transforms/externalLibsDexMerger/debug/0, ./android/app/build/intermediates/transforms/dexBuilder/debug/53.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/54.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/55.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/56.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/57.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/58.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/60.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/63.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/64.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/65.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/66.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/67.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/69.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/70.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/71.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/72.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/74.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/75.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/77.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/78.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/79.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/80.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/81.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/82.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/100.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/102.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/104.jar
        at com.android.builder.dexing.D8DexArchiveMerger.getExceptionToRethrow(D8DexArchiveMerger.java:124)
        at com.android.builder.dexing.D8DexArchiveMerger.mergeDexArchives(D8DexArchiveMerger.java:109)
        at com.android.build.gradle.internal.transforms.DexMergerTransformCallable.call(DexMergerTransformCallable.java:101)
        at com.android.build.gradle.internal.transforms.DexMergerTransformCallable.call(DexMergerTransformCallable.java:36)
        at java.util.concurrent.ForkJoinTask$AdaptedCallable.exec(ForkJoinTask.java:1424)
        ... 56 more
Caused by: com.android.tools.r8.CompilationFailedException: Compilation failed to complete
        at com.android.tools.r8.utils.ExceptionUtils.withCompilationHandler(ExceptionUtils.java:76)
        at com.android.tools.r8.utils.ExceptionUtils.withD8CompilationHandler(ExceptionUtils.java:45)
        at com.android.tools.r8.D8.run(D8.java:67)
        at com.android.builder.dexing.D8DexArchiveMerger.mergeDexArchives(D8DexArchiveMerger.java:107)
        ... 59 more
Caused by: com.android.tools.r8.utils.AbortException
        at com.android.tools.r8.utils.Reporter.failIfPendingErrors(Reporter.java:77)
        at com.android.tools.r8.utils.Reporter.fatalError(Reporter.java:58)
        at com.android.tools.r8.utils.ExceptionUtils.withCompilationHandler(ExceptionUtils.java:67)
        ... 62 more



FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformDexArchiveWithDexMergerForDebug'.
> com.android.build.api.transform.TransformException: java.lang.RuntimeException: com.android.builder.dexing.DexArchiveMergerException: Error while merging dex archives: ./android/app/build/intermediates/transforms/dexBuilder/debug/83, ./android/app/build/intermediates/transforms/externalLibsDexMerger/debug/0, ./android/app/build/intermediates/transforms/dexBuilder/debug/53.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/54.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/55.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/56.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/57.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/58.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/60.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/63.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/64.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/65.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/66.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/67.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/69.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/70.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/71.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/72.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/74.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/75.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/77.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/78.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/79.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/80.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/81.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/82.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/100.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/102.jar, ./android/app/build/intermediates/transforms/dexBuilder/debug/104.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 9s
159 actionable tasks: 94 executed, 65 up-to-date
Could not install the app on the device, read the error above for details.
Make sure you have an Android emulator running or a device connected and have
set up your Android development environment:
https://facebook.github.io/react-native/docs/getting-started.html

```

`$ git diff`

```diff
diff --git a/android/app/build.gradle b/android/app/build.gradle
index d0e6be4..8df4a65 100644
--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -116,7 +116,7 @@ android {
 
     defaultConfig {
         applicationId "com.example.myapp"
-        minSdkVersion 16
+        minSdkVersion 19
         targetSdkVersion 26
         versionCode 10
         versionName "1.0.0"
@@ -187,6 +187,7 @@ repositories {
 
 
 dependencies {
+    compile project(':react-native-view-pdf')
     compile project(':react-native-fs')
     compile project(':react-native-default-preference')
     compile project(':react-native-iap')
diff --git a/android/app/src/main/java/com/example/myapp/android/MainApplication.java b/android/app/src/main/java/com/example/myapp/android/MainApplication.java
index 2dda201..4970014 100644
--- a/android/app/src/main/java/com/example/myapp/android/MainApplication.java
+++ b/android/app/src/main/java/com/example/myapp/android/MainApplication.java
@@ -3,6 +3,7 @@ package com.example.myapp;
 import android.app.Application;
 
 import com.facebook.react.ReactApplication;
+import com.reactlibrary.PDFViewPackage;
 import com.rnfs.RNFSPackage;
 import com.reactlibrary.RNDefaultPreferencePackage;
 import com.dooboolab.RNIap.RNIapPackage;
@@ -35,6 +36,7 @@ public class MainApplication extends Application implements ReactApplication {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
+            new PDFViewPackage(),
             new RNFSPackage(),
             new RNDefaultPreferencePackage(),
             new RNIapPackage(),
diff --git a/android/settings.gradle b/android/settings.gradle
index c7ba35a..5eb0511 100644
--- a/android/settings.gradle
+++ b/android/settings.gradle
@@ -1,4 +1,6 @@
 rootProject.name = 'MyApp'
+include ':react-native-view-pdf'
+project(':react-native-view-pdf').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-view-pdf/android')
 include ':react-native-fs'
 project(':react-native-fs').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-fs/android')
 include ':react-native-default-preference'
diff --git a/ios/MyApp.xcodeproj/project.pbxproj b/ios/MyApp.xcodeproj/project.pbxproj
index 458e05e..0786461 100644
--- a/ios/MyApp.xcodeproj/project.pbxproj
+++ b/ios/MyApp.xcodeproj/project.pbxproj
@@ -59,6 +59,7 @@
 		D117CAC99F7940938AED6E5A /* FontAwesome5_Brands.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 055C61CD684B4254BFC81CC2 /* FontAwesome5_Brands.ttf */; };
 		2D703F5435BF4D7CA0A03D17 /* FontAwesome5_Regular.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 0E07CA114E0B41579044A944 /* FontAwesome5_Regular.ttf */; };
 		E36661E85A144B46AFC5EF6F /* FontAwesome5_Solid.ttf in Resources */ = {isa = PBXBuildFile; fileRef = E3AF1B17206F4706B32089C5 /* FontAwesome5_Solid.ttf */; };
+		AA3E6633F1A541C687A1D25B /* libReactNativeViewPDF.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 31873E6001664FDB85615BE0 /* libReactNativeViewPDF.a */; };
 /* End PBXBuildFile section */
 
 /* Begin PBXContainerItemProxy section */
@@ -371,6 +372,8 @@
 		055C61CD684B4254BFC81CC2 /* FontAwesome5_Brands.ttf */ = {isa = PBXFileReference; name = "FontAwesome5_Brands.ttf"; path = "../node_modules/react-native-vector-icons/Fonts/FontAwesome5_Brands.ttf"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = unknown; explicitFileType = undefined; includeInIndex = 0; };
 		0E07CA114E0B41579044A944 /* FontAwesome5_Regular.ttf */ = {isa = PBXFileReference; name = "FontAwesome5_Regular.ttf"; path = "../node_modules/react-native-vector-icons/Fonts/FontAwesome5_Regular.ttf"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = unknown; explicitFileType = undefined; includeInIndex = 0; };
 		E3AF1B17206F4706B32089C5 /* FontAwesome5_Solid.ttf */ = {isa = PBXFileReference; name = "FontAwesome5_Solid.ttf"; path = "../node_modules/react-native-vector-icons/Fonts/FontAwesome5_Solid.ttf"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = unknown; explicitFileType = undefined; includeInIndex = 0; };
+		02C2511A353945C99D2F0CD1 /* ReactNativeViewPDF.xcodeproj */ = {isa = PBXFileReference; name = "ReactNativeViewPDF.xcodeproj"; path = "../node_modules/react-native-view-pdf/ReactNativeViewPDF.xcodeproj"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = wrapper.pb-project; explicitFileType = undefined; includeInIndex = 0; };
+		31873E6001664FDB85615BE0 /* libReactNativeViewPDF.a */ = {isa = PBXFileReference; name = "libReactNativeViewPDF.a"; path = "libReactNativeViewPDF.a"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = archive.ar; explicitFileType = undefined; includeInIndex = 0; };
 /* End PBXFileReference section */
 
 /* Begin PBXFrameworksBuildPhase section */
@@ -407,6 +410,7 @@
 				115DA5B828BD4D419349BE53 /* libRNIap.a in Frameworks */,
 				141D61CE32774C5DA8BAD38B /* libRNDefaultPreference.a in Frameworks */,
 				9FC9DB9D14444816912E5CCE /* libRNFS.a in Frameworks */,
+				AA3E6633F1A541C687A1D25B /* libReactNativeViewPDF.a in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
@@ -612,6 +616,7 @@
 				6DAA896BC1F044B58EBFB4BD /* RNIap.xcodeproj */,
 				64862EFFEF5B4272B765D0E8 /* RNDefaultPreference.xcodeproj */,
 				0E72A35256BF4E20A4F1C6B8 /* RNFS.xcodeproj */,
+				02C2511A353945C99D2F0CD1 /* ReactNativeViewPDF.xcodeproj */,
 			);
 			name = Libraries;
 			sourceTree = "<group>";
@@ -1259,6 +1264,7 @@
 					"$(SRCROOT)/../node_modules/react-native-iap/ios",
 					"$(SRCROOT)/../node_modules/react-native-default-preference/ios",
 					"$(SRCROOT)/../node_modules/react-native-fs/**",
+					"$(SRCROOT)/../node_modules/react-native-view-pdf/ReactNativeViewPDF",
 				);
 				INFOPLIST_FILE = MyAppTests/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
@@ -1272,6 +1278,7 @@
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
+					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 				);
 				OTHER_LDFLAGS = (
 					"-ObjC",
@@ -1295,6 +1302,7 @@
 					"$(SRCROOT)/../node_modules/react-native-iap/ios",
 					"$(SRCROOT)/../node_modules/react-native-default-preference/ios",
 					"$(SRCROOT)/../node_modules/react-native-fs/**",
+					"$(SRCROOT)/../node_modules/react-native-view-pdf/ReactNativeViewPDF",
 				);
 				INFOPLIST_FILE = MyAppTests/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
@@ -1308,6 +1316,7 @@
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
+					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 				);
 				OTHER_LDFLAGS = (
 					"-ObjC",
@@ -1333,6 +1342,7 @@
 					"$(SRCROOT)/../node_modules/react-native-iap/ios",
 					"$(SRCROOT)/../node_modules/react-native-default-preference/ios",
 					"$(SRCROOT)/../node_modules/react-native-fs/**",
+					"$(SRCROOT)/../node_modules/react-native-view-pdf/ReactNativeViewPDF",
 				);
 				INFOPLIST_FILE = MyApp/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
@@ -1360,6 +1370,7 @@
 					"$(SRCROOT)/../node_modules/react-native-iap/ios",
 					"$(SRCROOT)/../node_modules/react-native-default-preference/ios",
 					"$(SRCROOT)/../node_modules/react-native-fs/**",
+					"$(SRCROOT)/../node_modules/react-native-view-pdf/ReactNativeViewPDF",
 				);
 				INFOPLIST_FILE = MyApp/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
@@ -1393,6 +1404,7 @@
 					"$(SRCROOT)/../node_modules/react-native-iap/ios",
 					"$(SRCROOT)/../node_modules/react-native-default-preference/ios",
 					"$(SRCROOT)/../node_modules/react-native-fs/**",
+					"$(SRCROOT)/../node_modules/react-native-view-pdf/ReactNativeViewPDF",
 				);
 				INFOPLIST_FILE = "MyApp-tvOS/Info.plist";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
@@ -1405,6 +1417,7 @@
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
+					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 				);
 				OTHER_LDFLAGS = (
 					"-ObjC",
@@ -1438,6 +1451,7 @@
 					"$(SRCROOT)/../node_modules/react-native-iap/ios",
 					"$(SRCROOT)/../node_modules/react-native-default-preference/ios",
 					"$(SRCROOT)/../node_modules/react-native-fs/**",
+					"$(SRCROOT)/../node_modules/react-native-view-pdf/ReactNativeViewPDF",
 				);
 				INFOPLIST_FILE = "MyApp-tvOS/Info.plist";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
@@ -1450,6 +1464,7 @@
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
+					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 				);
 				OTHER_LDFLAGS = (
 					"-ObjC",
@@ -1485,6 +1500,7 @@
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
+					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = "com.facebook.REACT.MyApp-tvOSTests";
 				PRODUCT_NAME = "$(TARGET_NAME)";
@@ -1516,6 +1532,7 @@
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 					"\"$(SRCROOT)/$(TARGET_NAME)\"",
+					"\"$(SRCROOT)/$(TARGET_NAME)\"",
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = "com.facebook.REACT.MyApp-tvOSTests";
 				PRODUCT_NAME = "$(TARGET_NAME)";
diff --git a/package.json b/package.json
index 7ec77ab..50a1523 100644
--- a/package.json
+++ b/package.json
@@ -66,6 +66,7 @@
     "react-native-iap": "1.2.4",
     "react-native-modal-overlay": "^1.2.3",
     "react-native-vector-icons": "^5.0.0",
+    "react-native-view-pdf": "^0.3.5",
     "react-navigation": "^1.0.0-beta.21",
     "realm": "^2.0.7",
     "underscore": "^1.8.3"
diff --git a/yarn.lock b/yarn.lock
index c5273cd..62a3dbb 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -7707,6 +7707,10 @@ react-native-vector-icons@^5.0.0:
     prop-types "^15.5.10"
     yargs "^8.0.2"
 
+react-native-view-pdf@^0.3.5:
+  version "0.3.5"
+  resolved "https://registry.yarnpkg.com/react-native-view-pdf/-/react-native-view-pdf-0.3.5.tgz#9a7b2d9bdb775a922954f6e6aef657160f17471f"
+
 react-native@^0.55.0:
   version "0.55.4"
   resolved "https://registry.yarnpkg.com/react-native/-/react-native-0.55.4.tgz#eecffada3750a928e2ddd07cf11d857ae9751c30"
```

**Comment by rumax** [comment]
In the log I can see:

```
Could not install the app on the device, read the error above for details.
Make sure you have an Android emulator running or a device connected and have
set up your Android development environment:
https://facebook.github.io/react-native/docs/getting-started.html
```

Also can you confirm that you can compile the project without `react-native-view-pdf`? Just after you changed the minSdkVersion.

**Comment by rumax** [comment]
@bulats Thanks a lot! Will change the name.

**Comment by rumax** [comment]
Fixed in NPM version 0.5.0
